docs/root/api/starting_envoy.rst

```diff
@@ -174,7 +174,7 @@ Specify the rate at which Envoy Mobile should flush its queued stats.
 
 Specifies the length of time a stream should wait without a headers or data event before timing out.
 Defaults to 15 seconds.
-See `the Envoy docs <https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#envoy-v3-api-field-extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-stream-idle-timeout>`_
```

Comment:
Is this deliberate?

Reply:
not sure why @buildbreaker updated. But still seems to generate correctly.